lib/minify.js

```diff
@@ -37,6 +38,9 @@ const minifiers = {
   },
   'esbuild': source => {
     return esbuild.transformSync(source, { loader: 'css', minify: true }).code;
+  },
+  '@parcel/css': source => {
+    return parcelCss.transform({filename: "", code: Buffer.from(source), minify: true}).code;
```

Comment:
If you want to get even smaller output, you can specify browser targets. This allows `@parcel/css` to e.g. remove vendor prefixes that aren't needed. See [here](https://github.com/parcel-bundler/parcel-css/blob/master/test.js#L11) for an example. Maybe it could be a separate column for that similar to the "advanced" modes of the other tools if you want to keep it separate. However, unlike those advanced modes, removing prefixes is a safe transform.

Reply:
Yes I copied from the example benchmark you have in the repository. I wanted the default entry in the benchmark to be similar to other tools as they don't have any browser-specific optimisation and thus I didn't know which browsers would be "reasonabe" to set.

Reply:
Sure. I definitely think it should be clearer from the benchmark results that many of the smallest sizes shown are using unsafe transforms though. Maybe something to work on separately.